### DISCUSSION TITLE
MacBook 体验有感 | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#318] [General]
# MacBook 体验有感 | 愧怍

作者是一位从来没用过苹果产品的程序员，但在使用了一周的 MacBook Pro 14 寸后，便爱不释手。

https://kuizuo.cn/blog/macbook-experience/

**Comment by pengweizhong** [comment]
破Mac，丝毫不重视用户体验，买回一年了还不适应，各种反人类操作，不明白到处宣扬mac好的目的是什么，windows难用已经时过去式了

**Reply by kuizuo** [reply]
已用 Mac 一年，表示不会再使用回 Windows，从果黑变成果粉，过几天准备换 IPhone 16 了。

**Reply by pengweizhong** [reply]
你现在用的哪个Mac？

**Reply by pengweizhong** [reply]
是14寸的m2pro吗

**Reply by kuizuo** [reply]
M2 Pro 丐版，被果子背刺了，买后 3 个月出 M3了😅

**Reply by pengweizhong** [reply]
咱俩同一时期买的，我的是m2max，32G，1Tb。看您很喜欢mac，咱俩交易一下如何？对换一下？这让我放着吃灰也能好受点😔

**Reply by kuizuo** [reply]
我也想换，16G 内存根本不够搞开发，但奈何这里面的代码在我短期内还无法迁移到其他设备中😔
![image](https://github.com/user-attachments/assets/ef9ad239-69ac-4fd8-850f-bd3c3fb44a65)

**Reply by pengweizhong** [reply]
我不着急，你确定换的话，咱俩加一下微信，等你弄好后再换

**Reply by kuizuo** [reply]
微信号 kuizuo12 有时间可以闲聊哈

**Comment by cc123hh** [comment]
之前也是苹果小黑子，直到买了一个air M4，香的不行，Windows已经太卡了，已经再也不想用了（
